### ISSUE TITLE
Can't align leadingHeaderBuilder towards bottom

### ISSUE THREAD

**Issue by raLaaaa** [JonasWanke/timetable#49]
Hey there,

I'm currently trying to create a `leadingHeaderBuilder` how ever I'm struggling with formatting it correctly.
To be more precise:

![image](https://user-images.githubusercontent.com/12799722/93534846-227f0c80-f946-11ea-8dc4-3127fdfbf5ee.png)

Basically I want to add a divider to the bottom as I did for the `dateHeaderBuilder` how ever no matter what I do the widget I add stays at the center of the left box. Only if I add a column with multiple widgets the box get filled. I was playing a little bit around and this is my current code which should work in my opinion:

```dart
leadingHeaderBuilder: (context, date) =>  Column(
	mainAxisAlignment: MainAxisAlignment.end,
	crossAxisAlignment: CrossAxisAlignment.end,
        children: [Text("SSS"), _buildDivider()]
,),
```

At least that's similiar to the way I did for the `dateHeaderBuilder`.
I'm not sure whether this is a bug or whether I'm doing something wrong or if there is an easier way to do this?

Thanks!

------
Currently using 0.2.7 of timetable.
Settings of my devices

Name: Pixel_API_R
CPU/ABI: Google Play Intel Atom (x86)
Path: C:\Users.android\avd\Pixel_API_R.avd
Target: google_apis_playstore [Google Play] (API level R)
Skin: pixel_silver
SD Card: 512M
fastboot.chosenSnapshotFile:
runtime.network.speed: full
hw.accelerometer: yes
hw.device.name: pixel
hw.lcd.width: 1080
image.androidVersion.codename: R
hw.initialOrientation: Portrait
image.androidVersion.api: 29
tag.id: google_apis_playstore
hw.mainKeys: no
hw.camera.front: emulated
avd.ini.displayname: Pixel API R
hw.gpu.mode: auto
hw.ramSize: 1536
PlayStore.enabled: true
fastboot.forceColdBoot: no
hw.cpu.ncore: 3
hw.keyboard: yes
hw.sensors.proximity: yes
hw.dPad: no
hw.lcd.height: 1920
vm.heapSize: 228
skin.dynamic: yes
hw.device.manufacturer: Google
hw.gps: yes
hw.audioInput: yes
image.sysdir.1: system-images\android-R\google_apis_playstore\x86
showDeviceFrame: yes
hw.camera.back: virtualscene
AvdId: Pixel_API_R
hw.lcd.density: 420
hw.arc: false
hw.device.hash2: MD5:55acbc835978f326788ed66a5cd4c9a7
fastboot.forceChosenSnapshotBoot: no
fastboot.forceFastBoot: yes
hw.trackBall: no
hw.battery: yes
hw.sdCard: yes
tag.display: Google Play
runtime.network.latency: none
disk.dataPartition.size: 6442450944
hw.sensors.orientation: yes
avd.ini.encoding: UTF-8
hw.gpu.enabled: yes

**Comment by raLaaaa** [comment]
I was digging into to code of the `timetable_header` and adapted it slightly:

```dart
  @override
  Widget build(BuildContext context) {
    // Like [WeekYearRules.iso], but with a variable first day of week.
    final weekYearRule =
        WeekYearRules.forMinDaysInFirstWeek(4, controller.firstDayOfWeek);

    return Row(
      children: <Widget>[
            SizedBox(
            width: hourColumnWidth,
            height: context.timetableTheme?.totalDateIndicatorHeight ?? 72, // thats what i added
            child: ValueListenableBuilder<LocalDate>(
              valueListenable: controller.dateListenable,
              builder: (context, date, _) {
                final customHeader = leadingHeaderBuilder?.call(context, date);
                if (customHeader != null) {
                  return customHeader;
                }

                return Center(
                  child: WeekIndicator(weekYearRule.getWeekOfWeekYear(date)),
                );
              },
            ),
          ),
        Expanded(
          child: Column(
            mainAxisSize: MainAxisSize.min,
            children: <Widget>[
              SizedBox(
                height: context.timetableTheme?.totalDateIndicatorHeight ?? 72,
                child: MultiDateHeader(
                  controller: controller,
                  builder: dateHeaderBuilder,
                ),
              ),
              AllDayEvents<E>(
                controller: controller,
                onEventBackgroundTap: onEventBackgroundTap,
                allDayEventBuilder: allDayEventBuilder,
              ),
            ],
          ),
        ),
      ],
    );
  }
```

It results in:

![image](https://user-images.githubusercontent.com/12799722/93575194-a1516500-f999-11ea-92a8-1c08024672b5.png)

Which is a little better  but not perfect if you are using all day events. Since we know the size of the `DateIndicator` we can add that. How ever what we are missing is the height of the `AllDayEvents`. Thats why we have an offset here. I tried to fix that using `IntrensicHeight` widget how ever it is not allowed to be used inside a `LayoutBuilder`. Any ideas how we could add the missing height of the `AllDayEvents`? 

Obviously if you have no `AllDayEvents` adding the height would work fine:

![image](https://user-images.githubusercontent.com/12799722/93575750-4c621e80-f99a-11ea-9cca-9b6ed0e9f0eb.png)

**Comment by JonasWanke** [comment]
I just found a workaround by avoiding the conflicting `LayoutBuilder` and am preparing a new release.

**Comment by JonasWanke** [comment]
This is now fixed as part of [v0.2.8](https://github.com/JonasWanke/timetable/releases/tag/v0.2.8).

**Comment by raLaaaa** [comment]
Thanks for the quick help!
